Basecamp.cpp

```diff
@@ -49,6 +49,27 @@ String Basecamp::_cleanHostname()
 	return clean_hostname;
 };
 
+/**
+ * Returns true if a secret for the setup WiFi AP is set
+ */
+bool Basecamp::isSetupModeWifiEncrypted(){
+	return (setupModeWifiEncryption_ == SetupModeWifiEncryption::secured);
+}
+
+/**
+ * Returns the SSID of the setup WiFi network
+ */
+String Basecamp::getSetupModeWifiName(){
+	return "ESP32_" + wifi.getHardwareMacAddress();
```

Comment:
To have a single point of truth, I would suggest to implement a `wifi.getApName()`-Getter in the wifiControl and return the info from there right here using `return wifi.getApName()`.